tests/UI/campaigns/data/faker/product.js

```diff
@@ -62,7 +62,7 @@ class ProductData {
     /** @type {string} Tac rule to apply the product */
     this.taxRule = productToCreate.taxRule || 'FR Taux standard (20%)';
 
-    /** @type {string} Specific price of the product */
+    /** @type {Object|{combinations: ?string, discount: ?number, startingAt: ?number}} Specific price of the product */
```

Comment:
https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

I'm not sure JSDoc got optional param notation

Wdyt about using the TypeScript syntax?
```suggestion
    /** @type {Object|{combinations?: string, discount?: number, startingAt?: number}} Specific price of the product */
```

Do you think that it looks clearer to read?

Reply:
From the official documentation https://jsdoc.app/tags-type.html , I think it's the correct syntax.
![image](https://user-images.githubusercontent.com/48441421/132376640-589213d8-3fc8-43d7-92b0-3f9ef5ceb736.png)

Reply:
Ok I missed it!